src/acorn/src/util.js

```diff
@@ -1,7 +1,3 @@
-export function isArray(obj) {
-  return Object.prototype.toString.call(obj) === "[object Array]"
-}
-
```

Comment:
I totally snuck in this minor change. It was only used in options.js above. We should kill util.js completely in a follow-up? :)